packages/plugins/jwt/src/index.ts

```diff
@@ -0,0 +1,184 @@
+import { JwksClient } from 'jwks-rsa'
+import jsonwebtoken, { JwtPayload, Algorithm } from 'jsonwebtoken'
+import { Plugin, createGraphQLError } from 'graphql-yoga'
+
+const { decode } = jsonwebtoken
+
+export interface JwtPluginOptions {
+  /**
+   * The endpoint to fetch keys from.
+   *
+   * For example: https://example.com/.well-known/jwks.json
+   */
+  jwksUri?: string
+  /**
+   * Signing key to be used to verify the token
+   * You can also use the jwks option to fetch the key from a JWKS endpoint
+   */
+  signingKey?: string
+  /**
+   * List of the algorithms used to verify the token
+   *
+   * Default: ["HS256"] (Allowed values: HS256, HS384, HS512, RS256, RS384, RS512, ES256, ES384, ES512, PS256, PS384, PS512, none)
+   */
+  algorithms?: Algorithm[]
+  /**
+   * The audience is the identifier of the API and is forwarded to your Auth API in order to specify for which API we are trying to authenticate our user for. E.g. if our API is hosted on `http://localhost:3000/graphql`, we would pass that value.
+   */
+  audience?: string
+  /**
+   * For example: `https://myapp.auth0.com/`
+   */
+  issuer: string
+  /**
+   * Once a user got successfully authenticated the authentication information is added on the context object under this field. In our resolvers we can then access the authentication information via `context._jwt.sub`.
+   */
+  extendContextField?: string
+  /**
+   * The name of the header to be used to extract the token from.
+   *
+   * Default: "Authorization"
+   */
+  headerName?: string
+  /**
+   * The type of the header
+   *
+   * Default: Bearer
+   */
+  headerType?: string
+  /**
+   * Function to extract the token from the request object
+   *
+   * Default: (request) => request.headers[headerName]
```

Comment:
```suggestion
   * Default: (request) => request.headers.get(headerName)
```

Reply:
But still it is more than `.get(headerName)` right? There is also `headerType` needs to be removed from the beginning.

Reply:
Yes, I don't know how to represent it in the documentation ><

Reply:
I think we can just mention about the default one that uses header type and name. What do you think?

Reply:
Ok, like in plain english instead of code